corehq/apps/api/resources/v0_5.py

```diff
@@ -222,10 +222,18 @@ def get_resource_uri(self, bundle_or_obj=None, url_name='api_dispatch_detail'):
                                                           pk=obj._id))
 
     def obj_create(self, bundle, **kwargs):
+        try:
+            normalized_username = normalize_username(bundle.data['username'], domain=kwargs['domain'])
+        except ValidationError as e:
+            raise BadRequest(str(e))
+
+        if user_exists(normalized_username).exists:
+            raise BadRequest(f'Username {normalized_username} already exists.')
```

Comment:
Should these error messages be translated?

Reply:
That's a good point. I translated the error message if a user already exists in 7fee083ba2f5e9d36b5f69ee530407d0cb7f32a5, but held off the validation error message. The ValidationError could be raised from different spots, and from what I can see are not translated. I don't want to add a generic translated error message here, because I think a more specific untranslated error message is more helpful than a translated generic error message. Definitely something that could be improved though! Let me know if you think this should be a blocker.

Reply:
I think it's fine to merge this as is since it's an improvement. Fixing translations can be done in a follow-up PR.